oak-run/src/main/java/org/apache/jackrabbit/oak/run/DataStoreOptions.java

```diff
@@ -76,6 +79,11 @@ public DataStoreOptions(OptionParser parser) {
         verbose =
             parser.accepts("verbose", "Option to get all the paths and implementation specific blob ids");
 
+        // Option NOT available for garbage collection operation - we throw an exception if both --collect-garbage and
+        // --verboseRootPath are provided in the command.
+        verboseRootPath = parser.accepts("verboseRootPath",
+                "Root path to output backend formatted ids/paths").availableUnless(collectGarbage).availableIf(verbose).withRequiredArg().withValuesSeparatedBy(DELIM).ofType(String.class);;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An empty statement (semicolon) not part of a loop](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5026544)